src/create-component-store.js

```diff
@@ -1,23 +1,32 @@
 import { useMemo, useState, useRef, useEffect } from 'react';
+import produce from 'immer-peasy';
 import createStore from './create-store';
 
 /**
  * Some good references on the topic of reinitialisation:
  * - https://github.com/facebook/react/issues/14830
  */
 
-export default function createComponentStore(model, config) {
-  return function useLocalStore(initialData) {
+export default function createComponentStore(model, config = {}) {
+  return function useLocalStore(initialData, injections = {}) {
```

Comment:
The same comments apply here as I made on the context store. We should consider refactoring this to allow a "runtime" config to be provided.

Reply:
same answers here :P